doc/Makefile

```diff
@@ -50,6 +50,8 @@ clean:
 
 .PHONY: html
 html:
+	rm -rf build
```

Comment:
do we really need this here? how was it working before?

Reply:
Don't need this, no... it was just a safety check for me when building locally that each new build was completely splatting the old...